harness/determined/cli/job.py

```diff
@@ -0,0 +1,109 @@
+import json
+from argparse import Namespace
+from typing import Any, List
+
+import yaml
+
+from determined.cli import render
+from determined.cli.session import setup_session
+from determined.common.api import authentication
+from determined.common.api.b import get_GetJobs
+from determined.common.declarative_argparse import Arg, Cmd, Group
+
+
+@authentication.required
+def ls(args: Namespace) -> None:
+    response = get_GetJobs(
+        setup_session(args),
+        resourcePool=args.resource_pool,
+        pagination_limit=args.limit,
+        pagination_offset=args.offset,
+        orderBy=args.order_by,
+    )
+    if args.yaml:
+        print(yaml.safe_dump(response.to_json(), default_flow_style=False))
+    elif args.json:
+        print(json.dumps(response.to_json(), indent=4, default=str))
+    elif args.table or args.csv:
+        headers = [
+            "Jobs Ahead",
+            "ID",
+            "Entity ID",
+            "Status",
+            "Type",
+            "Slots Acquired",
+            "Slots Requested",
+            "Name",
+            "User",
+            "Submission Time",
+        ]
+        values = [
+            [
+                j.summary.jobsAhead
+                if j.summary is not None and j.summary.jobsAhead > -1
+                else "N/A",
+                j.jobId,
+                j.entityId,
+                j.summary.state if j.summary is not None else "N/A",
+                j.type,
+                j.allocatedSlots,
+                j.requestedSlots,
+                j.name,
+                j.username,
+                j.submissionTime,
+            ]
+            for j in response.jobs
+        ]
+        render.tabulate_or_csv(headers, values, as_csv=args.csv)
+    else:
+        raise ValueError(f"Bad output format: {args.output}")
+
+
+pagination_args = [
+    Arg(
+        "--offset",
+        type=int,
+        default=0,
+        help="Offset the returned set.",
+    ),
+    Arg(
+        "--limit",
+        type=int,
+        default=50,
+        help="Limit the returned set.",
+    ),
+    Arg(
+        "--order-by",
+        type=str,
+        default="ORDER_BY_DESC",
```

Comment:
this might be confusing for kubernetes users, where higher priority is better. I don't think it's a major issue though and I'm happy to keep it like this for now.

Reply:
for this one I was trying to arrive at a set of arguments that we can reuse whereever pagination is involved. maybe a more universally applicable argument could be just `--reverse` ?